components/RadioButtons.tsx

```diff
@@ -33,7 +33,7 @@ const RadioButtons = ({
 
   return (
     <>
-      {title && <h3>{title}</h3>}
+      {title && <h3 className={styles.title}>{title}</h3>}
```

Comment:
Just as heads up for a11y heading hierarchy, my PR will [change the `h3` to an `h2`](https://github.com/ibi-group/survey/pull/17/files#diff-5bc2b5087dcba0ccdfe1abf46b3613772b9d3b43eba6b8a748bc3eda588cd3c3R42-R51). 
The css h3 selectors should also be changed to h2.